torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -1169,6 +1169,63 @@ def test_backward_autograd_engine_error(self):
                 # Run backwards, and validate we receive an error.
                 dist_autograd.backward(context_id, [val.sum()])
 
+    @dist_init
```

Comment:
Can we remove this hack since we have the proper fix now: https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/distributed/rpc/dist_autograd_test.py#L1558? I guess removing rpc.shutdown() and sys.exit() from that test should work. Would be nice to run that test 100 times just to make sure its not flaky after those changes.